### PR TITLE
Adding comments for admin.projectManagement.

### PR DESCRIPTION
**Pull request by egilmorez**
These would be the last comments to add before generating the reference.

Staged internally at https://firebase.devsite.corp.google.com/docs/reference/admin/node/admin.projectManagement

Thanks!

### CODE REVIEW

**Comment by hiranya911** [comment]
@nbegley can you take a look? We are moving our API references from the internally maintained externs to the the public GitHub repo.

**Comment by egilmorez** [comment]
Sounds good Hiranya!

Nick, this should be just a straight port of the comments from firebase-project-management-externs.js.  Thanks!

File: src/index.d.ts [hunk 1]
```diff
@@ -4786,56 +4786,349 @@ declare namespace admin.instanceId {
...
+     *
+     * @example
+     * ```javascript
+     * var resourceName = androidAppMetadata.resourceName;
+     * ```
+     */
     resourceName: string;
+
+    /**
+     * The optional user-assigned display name of the app.
```
[the remark this answers is not in the document]
Comment (egilmorez) [reply src/index.d.ts]: Good catch.

File: src/index.d.ts [hunk 2]
```diff
@@ -4786,56 +4786,349 @@ declare namespace admin.instanceId {
...
+     */
     getMetadata(): Promise<admin.projectManagement.AndroidAppMetadata>;
+
+    /**
+     * Sets the optional user-assigned display name of the app.
+     *
+     * @param newDisplayName The new display name to set.
+     *
+     * @return A promise that resolves when the display name has
+     *     been set.
```
[the remark this answers is not in the document]
Comment (egilmorez) [reply src/index.d.ts]: Hiranya, how strict do we want to be about that?  My preference would be to do that kind of cleanup in a follow-on PR.

[the remark this answers is not in the document]
Comment (hiranya911) [reply src/index.d.ts]: Yeah, it's ok to do that in a future PR.

**Pull request merged**